criteria/code-in-the-open.md

```diff
@@ -27,16 +27,13 @@ These aspects together create more opportunities for citizens to understand how
 ## What this does not do
 
 * Make source code or policy reusable.
-* Make the codebase and the code within it understandable to as many people as possible.
+* Make the [codebase](../glossary.md#codebase) and the code within it understandable to as many people as possible.
 
 ## How to test
 
-The source for any version currently in use is published on the internet where it can be seen:
-
-* from outside the original contributing organization
-* without the need for any form of authentication or authorization
-
-Review the codebase files and commit history to verify no sensitive information is included.
+* Confirm that the source for each version currently in use is published on the internet where it can be seen from outside the original contributing organization and without the need for any form of authentication or authorization.
```

Comment:
The original here had 2 bullets because it's a terribly long sentence otherwise. What was the logic for making it a single sentence?